docs/node-operators/getting-started.mdx

```diff
@@ -82,21 +89,27 @@ Instead, use the Docker instructions or build from source.
 
 ### Docker
 
-Docker is cross-platform tool that allows you to run a virtual machine that will execute any needed software. Follow the official installation instructions for your platform. See [Get Docker](https://docs.docker.com/get-docker/). After Docker is installed, proceed to the Mina Docker instructions on [Connect to the Network](connecting-to-the-network).
+Docker is a cross-platform tool. You can download and install Docker on multiple platforms. 
+
+Use the Docker installation instructions for your platform on [Get Docker](https://docs.docker.com/get-docker/). After Docker is installed, follow the [Connect to the Mina Network](/node-operators/connecting-to-the-network#docker) instructions.
 
 ### Build from source
 
-If you're running another Linux distro or some version of macOS, you can [try building Mina from source code](https://github.com/MinaProtocol/mina/blob/master/README-dev.md#building-mina). Please note that other operating systems haven't been tested thoroughly, and might have issues. Feel free to share any logs and get troubleshooting help in the [Mina Protocol Discord server](https://discord.gg/minaprotocol).
+On Linux or macOS operating systems, you can experiment with [Building Mina](https://github.com/MinaProtocol/mina/blob/master/README-dev.md#building-mina) from source code.
+
+Other operating systems haven't been tested thoroughly and might have issues. You can share logs and get troubleshooting help in the [Mina Protocol Discord server](https://discord.gg/minaprotocol).
+
+## Set up port forwarding and firewalls
 
-## Set up port forwarding and any firewalls
+If you are running a firewall, you must allow traffic on TCP port 8302.
```

Comment:
we repeat 8302 port ... is this because firewall is different than port forwarding? (forgive my ignorance)